<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DTablaCrear</class>
 <widget class="QDialog" name="DTablaCrear">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lNombre"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lLogin"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Contrase√±a</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lContra">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Karma</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sKarma"/>
   </item>
   <item>
    <widget class="QPushButton" name="bGuardar">
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bCancelar">
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
